docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+
+#### Test the cloud function
+
+Now you can test the script:
+
+1. Navigate to the **TRIGGER** tab in the Google Cloud console.
+
+2. Click the copy button to copy the trigger URL.
+   Paste it into a new browser window and hit your **Enter** key.
+   The browser should display "Notification sent."
+   If it displays an error, double-check your code, and double-check your [access authorization](https://cloud.google.com/functions/docs/securing/managing-access-iam).
+
+3. Check your email inbox.
+
+   {{<imgproc src="/tutorials/helmet/email-received.png" resize="x600" declaredimensions=true alt="The email, opened in a web email client." style="max-width: 400px" >}}
+
+   If you don't see an email, check your spam folder.
+   If you still don't see an email, make sure your SendGrid account is fully set up (2FA enabled, email confirmed) and that your email API key is [correctly configured](#configure-email-credentials).
+
+### Configure a webhook on your machine
+
+Now it's time to configure a [webhook](/build/configure/#webhooks) on your machine to trigger the email cloud function when a person is not wearing a hard hat.
+Since you configured data to sync only when an image of a person without a hard hat is captured, configuring the webhook to trigger each time an image is synced to the cloud will produce the desired result.
+
+Configure a webhook as follows:
+
+1. Navigate to the **Config** tab of your machine.
+   Toggle the **Mode** to **Raw JSON**.
+
+2. Paste the following JSON template into your raw JSON config.
+   `"webhooks"` is a top-level section like `"components"`, `"services"`, or any of the other config sections.
+
+   ```json {class="line-numbers linkable-line-numbers"}
+     "webhooks": [
+       {
+         "url": "<Insert your own cloud function URL>",
+         "event": {
+           "attributes": {
+             "data_types": ["binary"]
+           },
+           "type": "part_data_ingested"
+         }
+       }
+     ]
+   ```
+
+3. Replace the `url` value with your cloud function URL.
+   You can get this URL by copying it from the **TRIGGER** tab in the cloud function console.
+   Once you've done this, the `url` line should resemble, for example, `"url": "https://us-east1-blah-blah-123456.cloudfunctions.net/hat-email"`.
+
+4. Click **Save config**.
+
+## Test the whole system
+
+You've built all the pieces of the system and connected them together.
+Now it's time to test the whole thing.
+
+Make sure `viam-server` is running on your machine.
+Position yourself, without a hard hat, in front of your camera.
+Wait a couple of minutes for the email to arrive in your inbox, then celebrate your success when it does!
+Great work!
```

Comment:
I think this is a little too many exclamation marks
```suggestion
Wait a couple of minutes for the email to arrive in your inbox. Congratulations, you've successfully built your hard hat monitor!
```

Reply:
```suggestion
Wait a couple of minutes for the email to arrive in your inbox.
Congratulations, you've successfully built your hard hat monitor!
```